packages/wallets/src/constants/constants.tsx

```diff
@@ -11,33 +11,38 @@ export const optionsAndMultipliersContent = [
         description: 'Options and multipliers trading platform.',
         icon: <IcAppstoreDerivTrader />,
         redirect: '/',
+        smallIcon: <IcAppstoreDerivTrader style={{ height: '3.2rem', width: '3.2rem' }} />,
```

Comment:
any way to avoid inline styling? that should be really the last ditch effort

Reply:
This is `React.ComponentType<React.SVGAttributes<SVGElement>>` `weight` and `height` can be passed directly, `rem` values might work

Reply:
could it be just moved to .css ? why inlining?